src/docfx/build/document/DocumentProvider.cs

```diff
@@ -114,6 +124,21 @@ public string GetOutputPath(FilePath path, IReadOnlyList<string> monikers)
                 HashUtility.GetMd5Guid($"{depotName}|{sitePath.ToLowerInvariant()}").ToString());
         }
 
+        private bool TryGetDocumentIdConfig(PathString path, out DocumentIdConfig result, out PathString remainingPath)
+        {
+            foreach (var (basePath, config) in _documentIdRules)
+            {
+                if (path.StartsWithPath(basePath, out remainingPath))
```

Comment:
>path.StartsWithPath(basePath, [](start = 20, length = 29)

can this case be handled?
"_op_documentIdPathDepotMapping": {
        "./": {
          "depot_name": "Azure.azure-documents"
        }
}